test/Horde/Yaml/LoaderTest.php

```diff
@@ -14,9 +14,12 @@
  * @package    Yaml
  * @subpackage UnitTests
  */
-class Horde_Yaml_LoaderTest extends PHPUnit_Framework_TestCase
+
+namespace Horde\Yaml;
+
+class LoaderTest extends \PHPUnit\Framework\TestCase
```

Comment:
This style is OK and I used it myself first, but today we looked into the "use" statement. 
I'd prefer 
```
namespace Horde\Yaml;
use \PHPUnit\Framework\TestCase
/** 
 * Class level phpdoc here
 */
class LoaderTest extends TestCase
```

The order would be
```
/**
 * FileLevelDocBlock
 */
// declare for strict types would go here
// Then the namespace foo stuff
// Then the use stuff
/**
 * ClassLevelDocBlock
 */
class foo extends bar implements baz
```
See PSR-12 for details